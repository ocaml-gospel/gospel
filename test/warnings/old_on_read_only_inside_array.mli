val old_on_read_only : 'a array -> 'a
(*@ r = old_on_read_only a
    ensures r = old a.(0) *)
