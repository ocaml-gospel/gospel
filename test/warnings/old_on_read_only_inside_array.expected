File "old_on_read_only_inside_array.mli", line 3, characters 20-25:
Warning
There is a reference to old `a`, but is it read only.
